src/overlays/ovl_i4/fox_bo.c

```diff
@@ -1,91 +1,2197 @@
+    } else {
+        func_8007A6F0(&actor->obj.pos, 0x29121007);
+    }
+
+    return true;
+}
+
+void func_i4_8018D7F0(Actor* actor) {
+    if (D_8015F924 != 0) {
+        actor->state = 1;
+    }
+
+    func_i4_8018CC60(actor);
+
+    if (actor->state == 0) {
+        func_i4_8018D4F0(actor);
+        func_i4_8018D584(actor);
+    } else {
+        actor->timer_0CA[0] = 0;
+        actor->info.bonus = 0;
+        actor->info.unk_1C = 0.0f;
+    }
+    actor->scale = -1.0f;
+}
+
+s32 func_i4_8018D874(s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3f* rot, void* this) {
+    Actor* actor = (Actor*) this;
+
+    RCP_SetupDL(&gMasterDisp, 0x1D);
+    if (((limbIndex == 1) || (limbIndex == 2)) && (actor->timer_0C6 & 1)) {
+        RCP_SetupDL(&gMasterDisp, 0x29);
+        gDPSetPrimColor(gMasterDisp++, 0, 0, 255, 64, 64, 255);
+    }
+    if (((limbIndex == 1) || (limbIndex == 2)) && ((actor->health >= 100) || (D_8015F924 != 0))) {
+        *dList = NULL;
+    }
+
+    return false;
+}
+
+void func_i4_8018D960(Actor* actor) {
+    Animation_GetFrameData(&D_600F3D8, 0, actor->vwork);
+    Animation_DrawSkeleton(3, &D_600F4A4, actor->vwork, func_i4_8018D874, NULL, actor, gCalcMatrix);
+}
+
+s32 func_i4_8018D9CC(void) {
+    s32 i;
+    f32 spA8[17];
+    f32 sp64[17];
+    s32 j;
+    Vtx* sp5C = SEGMENTED_TO_VIRTUAL(D_6011E28);
+    Vtx* sp58 = SEGMENTED_TO_VIRTUAL(D_600C0B8);
+
+    for (i = 0; i < 17; i++) {
+        if ((i == 0) || (i == 16)) {
+            spA8[i] = 0.0f;
+            sp64[i] = 0.0f;
+        } else {
+            spA8[i] = RAND_FLOAT_CENTERED(400.0f);
+            sp64[i] = RAND_FLOAT_CENTERED(400.0f);
+        }
+    }
+
+    for (i = 0; i < 34; i++) {
+        j = (sp5C[i].v.ob[2] + 200) / 25;
+        sp58[i].v.ob[0] = sp5C[i].v.ob[0] + spA8[j];
+        sp58[i].v.ob[1] = sp5C[i].v.ob[1] + sp64[j];
+    }
+    return 0;
+}
+
+s32 func_i4_8018DE8C(Boss* boss) {
+    Vec3s D_i4_8019EEF8[26] = { { 0xFFAF, 0x00DC, 0xFFFC }, { 0x027B, 0x0042, 0xFE96 }, { 0x02A5, 0x00E7, 0x0427 },
+                                { 0x0172, 0x0154, 0x028A }, { 0xFF41, 0x01DE, 0xFD98 }, { 0x001E, 0x00D2, 0xFDAF },
+                                { 0x0529, 0x0053, 0xFFFE }, { 0x01CA, 0x0154, 0x0206 }, { 0xFEC8, 0x01DE, 0xFDCF },
+                                { 0x0286, 0x0042, 0xFE96 }, { 0x0294, 0x00B7, 0xFBE4 }, { 0x0302, 0x00F8, 0x0037 },
+                                { 0xFE02, 0x021B, 0xFFB3 }, { 0x00A3, 0x00DE, 0x0063 }, { 0x0286, 0x003C, 0x014B },
+                                { 0xFE06, 0x01EA, 0xFBEF }, { 0x0309, 0x00F8, 0xFFB3 }, { 0xFE08, 0x0245, 0x004D },
+                                { 0x0034, 0x00D8, 0x028A }, { 0xFBBF, 0x027D, 0x0001 }, { 0x01E0, 0x00DE, 0xFDDA },
+                                { 0xFF05, 0x020C, 0x01FA }, { 0x027B, 0x0042, 0x0140 }, { 0xFE06, 0x01EE, 0x040A },
+                                { 0x0130, 0x00DE, 0xFDB9 }, { 0xFF69, 0x01F0, 0x0252 } };
```

Comment:
```suggestion
                                { 0x0130, 0x00DE, 0xFDB9 }, { 0xFF69, 0x01F0, 0x0252 }, };
```
Also these should be decimal.